xcube_stac/store.py

```diff
@@ -35,12 +41,14 @@
     JsonStringSchema
 )
 
-from .constants import DATASET_OPENER_ID
-from .opener import StacDataOpener
-from .stac import Stac
+from .constants import (
+    DATASET_OPENER_ID,
+    MIME_TYPES,
+    STAC_SEARCH_ITEM_PARAMETERS
+)
 
 
-class StacDataStore(StacDataOpener, DataStore):
+class StacDataStore(DataStore):
     """STAC implementation of the data store.
 
     Attributes:
```

Comment:
You've made the `url` and `data_id_delimiter` attributes private now, so remove their documentation from the docstring.

Reply:
`url` and `data_id_delimiter` are the Args to initiate the class. I would therefore like to keep the documentation. `Attributes` will be changed to `Args`. (Discussion with @forman)